app/uk/gov/hmrc/gform/views/form/form.scala.html

```diff
@@ -96,20 +118,26 @@
 
     @formWithCSRF(action = page.formAction, args = 'class -> "js-form", 'autocomplete -> "on", 'novalidate -> "novalidate", 'id -> "gf-form") {
 
+    <div class="no-display">
+        <!-- this is here for the sole purpose of making it the default submit button, when fields of group exist -->
+        @displaySaveContinueButton()
+    </div>
+
       @{page.snippets}
 
       @{page.hiddenFieldsSnippets}
 
-      <input type="hidden" id="gform-action" name="save" value="Continue" />
-
-      @if(shouldDisplayContinue) {
-        @govukButton(Button(content = Text(buttonLabel), value=Some("Continue")))
-      }
-      @if(page.renderComeBackLater) {
-        <p class="govuk-body">
-          <a href="#" id="saveComeBackLater" class="govuk-link">@messages("linkText.saveAndComeBackLater")</a>
-        </p>
-      }
+      <div class="govuk-button-group">
```

Comment:
"Save/Continue" and "Save And Come Back" are button groups, as per the GDS pattern